doc/modules/linear_model.rst

```diff
@@ -313,15 +314,75 @@ is correct, i.e. that the data are generated by this model.
 They also tend to break when the problem is badly conditioned
 (more features than samples).
 
-.. figure:: ../auto_examples/linear_model/images/sphx_glr_plot_lasso_model_selection_001.png
-    :target: ../auto_examples/linear_model/plot_lasso_model_selection.html
+.. figure:: ../auto_examples/linear_model/images/sphx_glr_plot_lasso_lars_ic_001.png
+    :target: ../auto_examples/linear_model/plot_lasso_lars_ic.html
     :align: center
     :scale: 50%
 
+Mathematical details
+""""""""""""""""""""
+
+The definition of AIC (and thus BIC) might differ in the literature. In this
+section, we give more information regarding the criterion computed in
+scikit-learn. The AIC criterion is defined as:
```

Comment:
Are we sure, all our instances of AIC and BIC follow this convention?

Reply:
AIC/BIC are used in lasso and Gaussian mixture.
The AIC in `GaussianMixture` is defined as:

```
return -2 * self.score(X) * X.shape[0] + 2 * self._n_parameters()
```

which is this definition.

Reply:
Reassuring;-)
Should we then link to this section in the user guide of `GaussianMixture`?

Reply:
As long as we write `\hat{y}` instead of `X\hat{coef}`, the formulation is quite general.